pandas/_libs/tslibs/period.pyx

```diff
@@ -2194,7 +2194,7 @@ cdef class _Period(ABCPeriod):
         return (Period, object_state)
 
     def strftime(self, fmt: str) -> str:
-        """
+        r"""
```

Comment:
Is there a reason this is needed? (the docs seem to render fine: https://pandas.pydata.org/docs/dev/reference/api/pandas.Period.strftime.html#pandas.Period.strftime)

Reply:
there are `\(` in there that are invalid escapes

Reply:
It might be needed for sphinx, because as shown, it renders good. Did you check that it still renders fine with this change?

Reply:
I suppose this is copied from the stdlib docs, and there those escape is also used: https://raw.githubusercontent.com/python/cpython/3.8/Doc/library/datetime.rst

Reply:
I mainly notice this because the syntax highlighter (sublime text) marks it

Reply:
@jorisvandenbossche do you feel strongly that this should be reverted?  i care about this less than the rest of the PR, so would be OK with that

Reply:
I don't care about whether this should be reverted or not, but *if* you change it, you need to build the docs to check that it actually works (because just based on guessing, I wouldn't know whether this change is correct or not)